### DISCUSSION TITLE
Visual regression testing for react-pdf

### DISCUSSION THREAD

**Discussion by TimKochDev** [diegomura/react-pdf#2749] [Show and tell]
Hey everyone, my apps heavily rely on this awesome react-pdf package. Since sometimes version updates unfortunately include breaking layout changes, I had to find a way to do visual regression testing for the PDFs I generate. IIn case you have similar needs, here is what I came up with:

https://gist.github.com/TimKochDev/fff7b39d374d20b9996bf8094be2510a

**Comment by theskillwithin** [comment]
I wonder if there's a playwright way to see if the text on the pdf matches one of my pages.  because we have a page and a generated pdf that we want to always have the same content

**Reply by theskillwithin** [reply]
interesting https://dev.to/ryanroselloog/verify-pdf-contents-using-playwright-and-pdf2json-1dob
